tests/functional/helper.js

```diff
@@ -197,6 +206,7 @@ module.exports = {
   },
 
   async setElementProperty(id, property, type, value) {
+    await this.waitForElement(id)
```

Comment:
this breaks the logic. `set` should be used only when the element already exists.

Reply:
the thought behind it, was to wait for element before setting it. what do you think?